backend/services/core/project_settings_service.py

```
@@ -137,6 +137,72 @@ def get_all_settings(self, project_id: str) -> Dict[str, Any]:
+            settings_file = self._get_settings_file_path(project_id)
+            
+            if not settings_file.exists():
+                logger.info(f"No settings file found for project {project_id}, no LLM model selected")
+                return None
+            
+            with open(settings_file, 'r', encoding='utf-8') as f:
+                settings = json.load(f)
+            
+            selected_model = settings.get('selected_llm_model_id')
+            logger.info(f"Retrieved selected LLM model '{selected_model}' for project {project_id}")
+            return selected_model
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)